### ISSUE TITLE
Dropbox 500 error 

### ISSUE THREAD

**Issue by umartoru** [GnuCash-Pocket/gnucash-android#69]
when i try to connect to dropbox from this app for backup i get transfered to chrome for allowing gnucash to access dropbox folder, when i press the allow button i get 500 error,  i dont know where to post this issue if you can help resolve this thanks.

**Comment by Kaligula0** [comment]
This looks like a Dropbox issue or connection issue, not related to app. Anybody can confirm?

**Comment by Kaligula0** [comment]
But I get the same result, so it's not a connection problem. I suppose Dropbox issue?

**Comment by umartoru** [comment]
thanks @Kaligula0 , i used the prevous version 2.4 that works fine , but when i switched to gnucash-pocket it stoped working. so i think it might be some thing with gnucash as dropbox work fine with other apps.

**Comment by Kaligula0** [comment]
Maybe it's connected with app's trust / signing. I think we don't have original author's (@codeuser) signature / certificate. But that's just a speculation, because I'm not familiar with that.
Of course this could be something that one of the commits broke. Can you try if the oldest now available built apk works with Dropbox? https://github.com/GnuCash-Pocket/gnucash-android/actions/runs/7439060062

**Comment by seanomat** [comment]
@Kaligula0 I was in brief contact with the original @codeuser summer last year, when you all started picking up, where he left off.. He told me, he would get in contact with you to facilitate a smooth transfer. Did this not happen? I do have the email address here that I used to reach him with, but I am reluctant posting it here.

**Comment by Kaligula0** [comment]
@seanomat, I don't think so. Can you please move this information to #8? The admins are watching that thread, and if choose they know more.

**Comment by djbrown** [comment]
@seanomat you mean the original @codinguser? 🤯

**Comment by seanomat** [comment]
@djbrown pretty sure, yes. He was there from the very beginning.
